src/test/java/hudson/tasks/junit/TestResultTest.java

```diff
@@ -153,16 +154,20 @@ public void testTestSuiteDistributedOverMultipleFilesIsCountedAsOne() throws IOE
     /**
      * A common problem is that people parse TEST-*.xml as well as TESTS-TestSuite.xml.
      * See http://jenkins.361315.n4.nabble.com/Problem-with-duplicate-build-execution-td371616.html for discussion.
+     * 
+     * UPDATE: de-duplication causes missing tests, and the original use case hasn't worked in several releases.
+     * Therefore, the de-duplication logic has been removed.
      */
+    @Test
     public void testDuplicatedTestSuiteIsNotCounted() throws IOException, URISyntaxException {
```

Comment:
shouldn't this be:
```suggestion
    public void testDuplicatedTestSuiteIsCounted() throws IOException, URISyntaxException {
```

or simply removed because it's no longer a valid case?

Reply:
FYI I removed it

Reply:
Yes, I'm okay with that. Thanks!